src/pages/resultsView/mutation/Mutations.tsx

```diff
@@ -238,6 +242,33 @@ export default class Mutations extends React.Component<
         }
     }
 
+    @autobind
+    @action
+    protected getDownloadData(dataType?: DataType): string {
+        var flatdata: any = '';
+        if (
+            this.selectedGene &&
+            this.props.store.getMutationMapperStore(this.selectedGene)
+        ) {
+            var mutationMapperStore: ResultsViewMutationMapperStore;
+            mutationMapperStore = this.props.store.getMutationMapperStore(
+                this.selectedGene
+            )!;
+            let mutations = mutationMapperStore.getMutationData();
+            let downloadData: any[] = [];
+            let sidToStudy = mutationMapperStore.studyIdToStudy.result!;
+            mutations.forEach(m => {
+                let study: any;
+                study = sidToStudy[m.studyId];
+                Object.assign(m, { StudyName: study.name });
+                downloadData.push(m);
+            });
+
+            flatdata = tsvFormat(downloadData);
+        }
```

Comment:
Let's move this into `ResultsViewMutationMapperStore` and just do this: 
```suggestion
            flatData = mutationMapperStore.getDownloadData();
        }
```

Reply:
yeah, since it's an abstract operation we're doing here, i agree it belongs in the store where it could be used elsewhere if necessary